guides/posts/developers-stories/post.md

```diff
@@ -136,11 +140,15 @@ API aggregation. The ability to call an API then act on the data to call another
 
 </CodeStackr>
 
+<Csaba>
```

Comment:
Why is `Csaba` a component?

Reply:
To render Csaba's stories in the tweet-like format.

![Screenshot 2022-04-25 at 4 52 03 PM](https://user-images.githubusercontent.com/53901972/165079439-940e12c2-6ee4-42c2-ae05-7bb1b7428490.png)

Reply:
This should be a `Tweet` component and the details of Csaba should be passed to it.

Reply:
Did you mean passing Csaba's or other developer's details as the `props` directly from the markdown file?

If so, it will be lots of work because I will have to pass RapidAPI details in all the thread's post.md files.